docs/understanding-airbyte/jobs.md

```diff
@@ -39,6 +39,16 @@ Note: When a source has passed all of its messages, the docker process should au
 
 See the [architecture overview](high-level-view.md) for more information about workers.
 
+## Worker parallelization
+Airbyte exposes the following environment variable to change the maximum number of each type of worker allowed to run in parallel. 
+Tweaking these values might help you run more jobs in parallel and increase the workload of your Airbyte instance: 
+* `MAX_SPEC_WORKERS`: Maximum number of *Spec* workers allowed to run in parallel.
+* `MAX_CHECK_WORKERS`: Maximum number of *Check connection* workers allowed to run in parallel.
+* `MAX_DISCOVERY_WORKERS`: Maximum number of *Discovery* workers allowed to run in parallel.
+* `MAX_SYNC_WORKERS`: Maximum number of *Sync* workers allowed to run in parallel.
+
+The current default value for these environment variables is currently set to **5**.
```

Comment:
I don't think this is the right place for this. The Worker Parallelization only works for Kubernetes deployment. Also already have one section about the topic: https://docs.airbyte.io/deploying-airbyte/on-kubernetes#increasing-job-parallelism

Reply:
Are you sure they are only working for Kubernetes? When I check the call stack of these env var they are used to set a temporal `WorkerOptions` in `airbyte-workers/src/main/java/io/airbyte/workers/WorkerApp.java`: 
```java
  private static WorkerOptions getWorkerOptions(final int max) {
    return WorkerOptions.newBuilder()
        .setMaxConcurrentActivityExecutionSize(max)
        .build();
  }
```
I am not sure this temporal configuration is exclusive to K8S.

Reply:
Yeah, I am sure it works for K8s but I was not able to get it working in docker.

Reply:
@harshithmullapudi this should work in docker too. Why do you say you aren't able to get this working?

Reply:
In k8s I played with these parameters a bit earlier and could find the right config to run about 2000 syncs (in 2 hours) with 50 parallel syncs (SUBMITTER_NUM_THREADS=50) with about 5 workers (with MAX_SYNC_WORKERS=20)

I tried doing the same with docker and I could only run 15 at max. Not sure if I am missing something

Reply:
do you remember what your submitter_num_threads and max_sync_workers vars were?

Reply:
ah. I think I got it. We also need to increase the ports right? I gave 60 ports in K8s but didn't change it in docker.

Reply:
ah I think it's because the max sync worker variable is not actually passed into the worker yamls.

Reply:
> Ah. I think we also need to increase the ports right? I gave 60 ports in K8s but didn't change it in docker.

Nope. This is only used in Kube. Docker doesn't need explicit port allocation.

Reply:
I'm going to fix this in https://github.com/airbytehq/airbyte/pull/9209. 

Will tag folks when I'm done. Apart from this change, what we have here is correct.